Sources/GeneralIOs/Commands/Font.swift

```diff
@@ -0,0 +1,274 @@
+        }
+        catch {
+            throw Error.somethingGoingWrong("write extension font", error.localizedDescription)
+        }
+
+        // Same thing
+        sleep(1)
+
+        do {
+            try projectService.addFile(targetName: targetName, filePath: Path(Constant.extensionFontPath))
+        }
+        catch {
+           throw Error.somethingGoingWrong("add file in target", error.localizedDescription)
+        }
+
+        do {
+            try projectService.write()
+            print("✨ \(green("Successfully")) completed added fonts... ✨")
+        }
+        catch {
+           throw Error.somethingGoingWrong("save changes", error.localizedDescription)
+        }
+    }
+
+    public init() {
+    }
+
+    private func addFontsInInfoPlistAndFolderFonts(with fonts: [FileInfo],
+                                                   appFonts: AEXMLElement,
+                                                   target: String) throws {
+        for newFont in fonts {
+            var isContains: Bool = false
+            for appFont in appFonts.children {
+                if appFont.value == newFont.url.lastPathComponent {
+                    isContains = true
+                    break
+                }
+            }
+            if isContains == false {
+                appFonts.addChild(.init(name: "string", value: newFont.url.lastPathComponent))
+                let destination = URL(fileURLWithPath: Constant.fontsFolderPath + "/" + newFont.url.lastPathComponent)
+                if fileHelper.fileManager.fileExists(atPath: destination.path) == false {
+                    do {
+                        try fileHelper.fileManager.copyItem(at: newFont.url, to: destination)
+                    }
+                    catch {
+                        throw Error.somethingGoingWrong("copy item", error.localizedDescription)
+                    }
+
+                    // Here setup sleep 1 second when copy item in directory after install target for file.
+                    // if sleep delete when target is not install for file because cannot execute asynchronously.
+                    sleep(1)
+
+                    do {
+                        try projectService.addFile(targetName: target, filePath: Path(destination.relativePath))
+                        print("🎉 \(green("Added font:")) \(newFont.url.lastPathComponent) ... 🎉")
+                    }
+                    catch {
+                        throw Error.somethingGoingWrong("add target", error.localizedDescription)
+                    }
+                }
+            }
+        }
+    }
+
+    private func fonts(in folder: URL) -> [FileInfo] {
+        guard let filesInfo = try? fileHelper.contentsOfDirectory(at: folder) else {
+            return []
+        }
+        return filesInfo.flatMap { file -> [FileInfo] in
+            if file.isDirectory {
+                return fonts(in: file.url)
+            }
+
+            let ext = file.url.pathExtension
```

Comment:
I dont like word reduction names
```suggestion
            let extension = file.url.pathExtension
```